### DISCUSSION TITLE
Adding Hamsters

### DISCUSSION THREAD

**Discussion by nmskhan** [CoBrALab/RABIES#302] [General]
Hi
Thank you for creating one platform for processing fMRI data for all rodents. My lab is interested in examining the functional connectivity of Syrian hamsters, and brain functional differences. The brain size of the hamster is more or less similar to the brain of rats and also identical to rat brain morphology. My questions are
Is there any way to incorporate the hamster brain or enable hamster brain MRI into the preprocessing/analysis pipelines?
I do have a hamster atlas in high resolution and also an atlas label file, the vascular mask for that. 

What else (files) are needed to do these? Is there any way to add this to your docker/singularity container? 
Some of the data I collected are in a different orientation (axial (hamster) vs coronal (as in your sample mice data test)). Do I have to reorient this to suit to anatomical atlas?

Please let me know
Thanks
Ned

**Comment by Gab-D-G** [comment]
Hi Ned,

Trying out RABIES on hamster data sounds awesome! I don't anticipate particular issues for this purpose. The main factor that will affect the quality of processing is the quality of your images and your atlas.

I would recommend running the software with Singularity or Docker, and then provide the path to your atlas files as detailed in the command line interface for preprocessing https://rabies.readthedocs.io/en/latest/running_the_software.html#command-line-interface:~:text=Template%20Files%3A%0A%20%20Specify%20commonspace%20template%20and%20associated%20mask/label%20files.%20By%20default%2C%20RABIES%0A%20%20provides%20the%20mouse%20DSURQE%20atlas%0A%20%20https%3A//wiki.mouseimaging.ca/display/MICePub/Mouse%2BBrain%2BAtlases. Then it should run smoothly, as with other rodent data.

Concerning image orientation, this may cause issues. If the orientation mismatch your atlas, it will cause errors. And if you images are in a different orientation, the automatically-generated reports may display different axes of the brain. I recommend trying out, as perhaps orientation will be adjusted automatically within RABIES, and let us know if you encounter issues.

Best,
Gabriel

**Reply by gdevenyi** [reply]
Orientation is assumed to be RAS everywhere, but if the user were to provide atlases/masks/etc in a different orientation and use them then RABIES would happily maintain that throughout. The only issue may be slice timing correction being wrong.

**Reply by nmskhan** [reply]
Thanks Gabriel and Devenyi. Let me check and I will let you know if I encounter any issues. I am still trying to learn and understand the docker based processing. Thanks
Ned
